src/styles/Company.css

```diff
@@ -0,0 +1,66 @@
+.company-field {
+  padding: 30px;
+  font-size: 16px;
+  font-weight: 800;
+  max-width: 50%;
+}
+
+.companies-container > div:nth-of-type(1) {
+  background-color: #ec4c8a;
+}
+
+.companies-container div:nth-of-type(2) {
+  background-color: #fc5192;
+}
+
+.companies-container div:nth-of-type(4) {
+  background-color: #ec4c8a;
+}
+
+.companies-container div:nth-of-type(3) {
+  background-color: #fc5192;
+}
+
+.companies-container div:nth-of-type(5) {
+  background-color: #ec4c8a;
+}
+
+.companies-container div:nth-of-type(6) {
+  background-color: #fc5192;
+}
+
+.companies-container div:nth-of-type(7) {
+  background-color: #fc5192;
+}
+
+.companies-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: 100vh;
+  font-family: Lato, serif;
+}
+
+.div-company {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  overflow: hidden;
+  text-decoration: none;
+  border: none;
+  text-align: initial;
+  color: black;
+}
+
+.name-market {
+  display: flex;
+  flex-direction: column;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* img {
+  max-width: 2em;
+} */
```

Comment:
**You did a great job @erickhansx.** 

_However, this issue counts as a required change._ 

- [ ] To make your code professional, kindly remove unused commented codes from your project :).

Reply:
Thank you for your feedback. Removed.